src/EditorFeatures/TestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs

```diff
@@ -37,6 +37,7 @@ public struct TestParameters
             internal readonly int index;
             internal readonly CodeActionPriority? priority;
             internal readonly bool retainNonFixableDiagnostics;
+            internal readonly bool shouldVerifyEntireSpan;
```

Comment:
>shouldVerifyEntireSpan [](start = 35, length = 22)

I think the flag name is misleading, especially given the default parameter value is false in the constructor, and might seem that we are skipping span verification by default. How about using an enum here:

```csharp
enum SpanVerificationKind
{
   Intersect,
   Match,
}
```

The default parameter value in constructor would be `SpanVerificationKind.Intersect`